numpy/_expired_attrs_2_0.py

```diff
@@ -20,9 +20,9 @@
     "NZERO": "Use `-0.0` instead.",
     "PZERO": "Use `0.0` instead.",
     "add_newdoc": 
-        "It's an internal function and doesn't have a replacement.",
+        "It's now available from `np.lib.add_newdoc`.",
     "add_docstring": 
-        "It's an internal function and doesn't have a replacement.",
+        "It's now available from `np.lib.add_docstring`.",
```

Comment:
Minor rephrase proposed (use "still ... as"), for both these messages: 
```
It's stilll available as `np.lib.add_docstring`.
```
That hints that it has always been there, not that it recently got moved there.

Reply:
That's right - I changed both lines.